backend/fms_core/tests/test_services/individual_test.py

```diff
@@ -0,0 +1,38 @@
+from django.test import TestCase
+
+from fms_core.services.individual import get_taxon
+
+class IndividualServicesTestCase(TestCase):
+
+    def test_get_taxon(self):
+        HOMO_SAPIENS_NAME = 'Homo sapiens'
+        HOMO_SAPIENS_NCBI_ID = 9606
+
+        # Check with normalized name
+        taxon, _, _ = get_taxon(name='Homo sapiens')
```

Comment:
You could use the constant as input too and make a different one for the non-normalized test.